server/core/store.go

```diff
@@ -114,16 +114,16 @@ func (s *StoreInfo) LeaderScore(delta int64) float64 {
 // RegionScore returns the store's region score.
 func (s *StoreInfo) RegionScore(highSpaceRatio, lowSpaceRatio float64, delta int64) float64 {
 	if s.RegionSize == 0 {
-		return float64(delta)
+		return float64(delta) / math.Max(s.RegionWeight, minWeight)
```

Comment:
Is it ok to return a negative score?

Reply:
when `region size == 0`, the store must selected as target, so the `delta` must larger than 0